style.css

```diff
@@ -2,229 +2,780 @@
+  @import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+  @import url('https://fonts.googleapis.com/css2?family=Crete+Round&display=swap');
+  @import url('https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap');
+
+  text-decoration: none;
+  list-style-type: none;
+  box-sizing: border-box;
 }
 
 body {
-  margin: 0px;
+  width: auto;
+  max-width: 100%;
+  height: auto;
+  overflow-x: hidden;
 }
 
-.section1 {
-  margin: 0px;
+/* start to homepage */
+.homepage-container {
+  display: grid;
+  grid-template-columns: 1349px;
+  justify-self: center;
+  height: 900px;
+  min-width: 1366px;
+  max-width: 100%;
+  background-color: #1c1a19;
+}
+
+.header {
   display: flex;
+  background-color: #3c3a39;
+  width: 1349px;
+  max-width: 100%;
+  height: 100px;
+  max-height: 100%;
+  z-index: 2;
+}
+
+.logo-text {
+  color: white;
+  margin-top: 30px;
+  margin-left: 150px;
+  font-family: 'Inter', sans-serif;
+  font-size: 26px;
+  font-weight: 900;
+  text-decoration: none;
+  display: inline-flex;
+  width: 100vh;
+}
+
+.navigation {
+  display: flex;
+  width: fit-content;
+  height: 100px;
+  align-items: center;
+  margin-left: 50px;
+  gap: 20px;
+}
+
+.nav-text {
+  font-family: 'Inter', sans-serif;
+  font-size: 16px;
+  color: white;
+  font-weight: 600;
+}
+
+.menu-bar {
+  display: none;
 }
 
 .section2 {
   margin: 0px;
+  padding: 0px;
+  overflow-y: hidden;
+  overflow-x: hidden;
+}
+
+.intro-text {
+  display: grid;
+  grid-template-columns: 683px;
+  row-gap: 15px;
+  margin-top: 80px;
+  margin-left: 150px;
+}
+
+.text1 {
+  color: #f55800;
+  font-size: 75px;
+  font-family: 'Crete Round', sans-serif;
+  font-weight: 400;
+}
+
+.text2 {
+  color: white;
+  font-size: 30px;
+  font-family: 'Inter', sans-serif;
+  font-weight: bold;
+}
+
+.text3 {
+  color: white;
+  font-size: 22px;
+  font-family: 'Inter', sans-serif;
+  width: 600px;
+}
+
+.bottom {
+  transform: translate(-14px, -223px);
+}
+
+.icons {
+  list-style-type: none;
+  display: inline-flex;
+  flex-direction: column;
+  gap: 10px;
+  width: fit-content;
+  align-items: center;
+}
+
+.image {
+  display: none;
+}
+
+.image-1 {
+  z-index: 1;
+  transform: translate(721px, -731px);
+}
+
+/* end to homepage */
+
+/* start to work section */
+.work-section-container {
+  margin-bottom: 50px;
+}
+
+.heading {
   display: flex;
+  gap: 10px;
+  margin-left: 155px;
+  margin-top: 70px;
+  width: fit-content;
+  height: 100px;
+  align-items: center;
 }
 
-.intro-container {
-  background-color: #1c1a19;
+.title-1 {
+  font-size: 35px;
+  font-weight: none;
+  color: #091e42;
+  font-family: 'crete round', sans-serif;
+  width: 294px;
+  margin: 0px;
+  padding: 0px;
+}
+
+.line-top {
+  width: 705px;
+  opacity: 40%;
+  margin-left: 20px;
+}
+
+.work-grid-container {
+  display: flex;
+  margin-left: 155px;
+  margin-top: 150px;
+  gap: 20px;
+}
+
+.grid-item-1 {
+  width: 520px;
   height: auto;
-  width: 105.5vw;
+}
```

Comment:
- [ ] I have noticed that you are using a fixed width. Due to this, the item can not be responsive and therefore it is causing a horizontal scroll. The traditional horizontal scrollbar burdens the user by requiring constant attention and greater physical effort to maintain the dragging 😞. Kindly fix this as it will be the step to solve the horizontal scroll effect in your portfolio ☺️.